.travis.yml

```diff
@@ -3,12 +3,12 @@ node_js:
 - '8'
 install:
 - bundle install
-- npm install
+- TMPDIR=/tmp yarn install
```

Comment:
`yarn` is available in travis by default?

Reply:
Yes, if you have a yarn.lock file https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn